dapp-developer-guide/tutorials/upload-your-file-folder-with-ethfs-cli.md

```diff
@@ -15,48 +16,97 @@ For the content to be uploaded, let's assume that
 
 If you have not already done so, you can install `ethfs-cli` using the following command:
 
-```
+```bash
 npm i -g ethfs-cli
 ```
+If you installed it a long time ago, please run the same command to update to the latest version.
 
 ## Step 2: Create a `FlatDirectory` Contract
 
 A [FlatDirectory](https://docs.web3url.io/advanced-topics/flatdirectory) contract serves as a container that needs to be created before uploading files. 
 
-The following command creates a `FlatDirectory` on the default chain (**Ethereum Mainnet**) using the private key `0x112233...`
+### Command Syntax
 
+```bash
+ethfs-cli create -p <private-key> -c <chain-id>
 ```
-$ ethfs-cli create -p 0x112233...
-```
+This command creates a `FlatDirectory` on a specified blockchain. You need to provide a private key and the blockchain's ID. 
 
-If you intend to create it on other chains that support EVM, you need to add the chain ID using the `-c` flag, for example, `11155111` for **Sepolia**.
+If the part `-c <chain-id>` is omitted, **Ethereum Mainnet** will be specified by default.
 
+### Example: SWC Beta
+
+```bash
+ethfs-cli create -p 0x112233... -c 3335
 ```
-$ ethfs-cli create -p 0x112233... -c 11155111
+
+You will get a `FlatDirectory` address after the transaction is confirmed on SWC beta testnet:
+
+```bash
+FlatDirectory: Address is 0xab351F35B82B20C1a253ae16523c5E2D60B56D6E
 ```
 
-You will get a `FlatDirectory` address after the transaction is confirmed:
+### Example: Sepolia
 
+```bash
+ethfs-cli create -p 0x112233... -c 11155111
 ```
-$ ethfs-cli create -p 0x112233... -c 11155111
-chainId = 11155111
-providerUrl = https://rpc.sepolia.org
+
+You will get a `FlatDirectory` address after the transaction is confirmed on Sepolia:
+
+```bash
 FlatDirectory: Address is 0x2f7696D4284358A2E8fDb4DF772dAd60c2c8fbAd
 ```
 
 ## Step 3: Upload Files
 
-In this section, you will upload the files/folder into the `FlatDirectory` that you just created using the following command:
+In this section, you will upload the files/folder into the `FlatDirectory` that you just created.
+
+### Command Syntax
 
+```bash
+ethfs-cli upload \
+    -f <directory|file> \
+    -a <address> \
+    -c <chain-id> \
+    -p <private-key> \
+    -t <upload-type>
 ```
-ethfs-cli upload -f <directory|file> -a <address> -c <chain-id> -p <private-key> -t <upload-type>
+You need to provide the contract address (`-a <address>`), blockchain ID (`-c <chain-id>`), private key (`-p <private-key>`), and the type of upload (`-t <upload-type>`).
```

Comment:
Is there such a term for _blockchain ID_? Not sure, network ID is more commonly used?